libs/cms/src/lib/cms.contentful.service.ts

```diff
@@ -155,6 +163,36 @@ export class CmsContentfulService {
     }
   }
 
+  async getOrganizationLogos(
+    organizationTitles: string[],
+  ): Promise<Array<string | null>> {
+    const params = {
+      ['content_type']: 'organization',
+      select: 'fields.logo,fields.title',
+      'fields.title[in]': organizationTitles.join(','),
```

Comment:
Does the library not support arrays? :(

I can't find great documentation around this but we might need to urlencode each title somehow ... or hope there's no organisation with "#" or "," in its name.

Reply:
What do you mean somehow? Are you thinking something else that this maybe
```
      'fields.title[in]': organizationTitles
        .map((title) => encodeURIComponent(title))
        .join(','),
```